spring-boot-autoconfigure/src/main/resources/META-INF/spring.factories

```diff
@@ -69,7 +69,8 @@ org.springframework.boot.autoconfigure.web.HttpMessageConvertersAutoConfiguratio
 org.springframework.boot.autoconfigure.web.MultipartAutoConfiguration,\
 org.springframework.boot.autoconfigure.web.ServerPropertiesAutoConfiguration,\
 org.springframework.boot.autoconfigure.web.WebMvcAutoConfiguration,\
-org.springframework.boot.autoconfigure.websocket.WebSocketAutoConfiguration
+org.springframework.boot.autoconfigure.websocket.WebSocketAutoConfiguration,\
+org.springframework.boot.autoconfigure.cxf.CXFServletAutoConfiguration
```

Comment:
We try to keep `spring.factories` ordered alphabetically so this new line should be earlier in the file